package.json

```diff
@@ -54,6 +54,10 @@
     "fuzzy": "^0.1.3",
     "lodash-es": "^4.17.21",
     "react-ace": "^11.0.1",
+    "react-markdown": "^9.0.1",
+    "react-mde": "^11.5.0",
+    "rehype-raw": "^7.0.0",
+    "remark-gfm": "^4.0.0",
```

Comment:
Are these libraries to support for html tags? If so, I'd rather we leave them out and now have support for HTML tags

Reply:
Because I'm concerned about XSS - https://www.npmjs.com/package/react-mde#xss-concerns

Reply:
My bad, I see we already support some tags in the form engine - https://github.com/openmrs/openmrs-esm-form-engine-lib/blob/810fd93961be3a38c3e3afe351996d676de65eec/src/components/inputs/markdown/markdown-wrapper.component.tsx#L9

Reply:
> Are these libraries to support for html tags? If so, I'd rather we leave them out and not have support for HTML tags

@NethmiRodrigo , we do need `remark-gfm` if we are to have **GitHub Flavored Markdown (GFM)** e.g tables, task lists, strikethroughs, Autolinks e.t.c

And we need The `rehypeRaw` plugin processes raw HTML embedded within Markdown content, allowing it to render as HTML elements in the final output.

Are there some of these formatting options we don't need so we can keep it as simple as possible?

Reply:
Lets limit the formatting options to only the ones that are *currently* supported by the react form engine. We don't need these libraries then right?

Reply:
> Lets limit the formatting options to only the ones that are _currently_ supported by the react form engine. We don't need these libraries then right?

We now have this implemented